posts/intro-to-mathport.md

```diff
@@ -0,0 +1,224 @@
+---
+author: 'Scott Morrison'
+category: 'mathport'
+date: 2021-12-11 12:41:00+00:00
+description: ''
+has_math: true
+link: ''
+slug: intro-to-mathport
+tags: ''
+title: Update on mathport (Dec 2021)
+type: text
+---
+
+# An update on mathport
```

Comment:
I think these should probably be `<h2>` (`##`) so as not to conflict with the article-level titles which are `<h1>`. [According to MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#usage_notes):

> The HTML specification includes the concept of an outline formed by the use of <section> elements. If this were implemented it would enable the use of multiple <h1> elements, giving user agents—including screen readers—a way to understand that an <h1> nested inside a defined section is a subheading. This functionality has never been implemented; therefore it is important to use your headings to describe the outline of your document.

Reply:
Actually, this particular heading should just be removed, it duplicates the automated one.